kubernetes/yamls/api-server/deployment.yml

```diff
@@ -18,7 +18,7 @@ spec:
         emptyDir: {}
       containers:
       - name: api-server
-        image: eventyay/nextgen-api-server:latest
```

Comment:
It is still nextgen-api-server at docker https://hub.docker.com/r/eventyay/nextgen-api-server/

@mariobehling @niranjan94 Can you please look into it.

Reply:
Should I change it back to `nextgen` ? @poush